### DISCUSSION TITLE
Why can only get access video with 127.0.0.1:8081?

### DISCUSSION THREAD

**Discussion by hwypengsir** [Motion-Project/motion#1704] [Q&A] [answered]
My local pc's address is `192.168.31.167`.

    sudo ifconfig | grep inet
            inet 192.168.31.167  netmask 255.255.255.0  broadcast 192.168.31.255

Install motion and start it:

    sudo apt install motion
    sudo motion

I can only get access video with `127.0.0.1:8081` in browser.

[![enter image description here][1]][1]

Why `192.168.31.167:8081` can't play the video?

  [1]: https://i.stack.imgur.com/7wWHz.png

**Comment by freekdk** [comment, marked as answer]
Op woensdag 23 augustus 2023 14:19:00 CEST schreef hwypengsir:
> My local pc's address is `192.168.31.167`.
> 
>     sudo ifconfig | grep inet
>             inet 192.168.31.167  netmask 255.255.255.0  broadcast
> 192.168.31.255
> 
> Install motion and start it:
> 
>     sudo apt install motion
>     sudo motion
> 
> I can only get access video with `127.0.0.1:8081` in browser.
> 
> [![enter image description here][1]][1]
> 
> Why `192.168.31.167:8081` can't play the video?
> 
>   [1]: https://i.stack.imgur.com/7wWHz.png

Because in /etc/motion.conf you did not change the line with stream_localhost 
to off. Also look at stream_auth_method.

-- 

vr.gr.

Freek de Kruijf
